.ui/pgWebView.pgx

```diff
@@ -81,6 +82,36 @@
 				"testId": "wvs2eJOv5",
 				"usePageVariable": true
 			}
+		},
```

Comment:
This was added by @cico643 Is this needed change?

Reply:
No, you can revert the changes on this page. @alimoncul